### PR TITLE
Fix stub for Yaf 

### PR DESCRIPTION
**Pull request by wxxiong6**
Fix stub for Yaf

### CODE REVIEW

**Comment by wxxiong6** [comment]
Fix stub for Yaf

File: yaf/yaf_namespace.php [hunk 1]
```diff
@@ -2282,22 +2282,19 @@ protected function setAllHeaders(array $headers)
     /**
      * @link https://secure.php.net/manual/en/yaf-response-abstract.getheader.php
      *
-     * @param string $name
-     *
      * @return mixed
      */
-    public function getHeader($name = null)
```
Comment (andrey-sokolov) [inline yaf/yaf_namespace.php]: Same here

File: yar/yar.php [hunk 2]
```diff
@@ -1,5 +1,30 @@
 <?php
 
+
+/**
+ * The constants below are defined by this extension,
+ * and will only be available when the extension has either been compiled into PHP or dynamically loaded at runtime.
+ * @link https://secure.php.net/manual/en/yar.constants.php
+ */
+define('YAR_VERSION', '	2.0.5', true);
```
Comment (andrey-sokolov) [inline yar/yar.php]: These constants are not case insensitive, so,please, get rid of the third argument

Comment (wxxiong6) [reply yar/yar.php]: I fixed the problem

**Comment by pestretsov** [comment]
Could you please fix the build so we can merge your changes?

**Pull request merged**